index.js

```diff
@@ -0,0 +1,107 @@
+var _ = require('lodash');
+var request = require('request');
+
+var secrets = require('./secret/secrets');
+var stations = require('./resources/stations');
+var utils = require('./lib/utils');
+
+var APP_ID = secrets.applicationId;
+var AlexaSkill = require('./lib/AlexaSkill');
+
+var wmataReq = request.defaults({
+  baseUrl: 'https://api.wmata.com',
+  qs: secrets.apiKey
+});
+
+var getWmataResponse = function(endpoint, response, callback) {
+  wmataReq(endpoint, function(error, res, body) {
+    if (!error && res.statusCode === 200) {
+      callback(JSON.parse(body));
+    } else {
+      response.tell('There was an error making the WMATA request, please try again later.');
```

Comment:
For now, this should be fine. Maybe later we can refactor this, so individual intents can handle errors with WMATA differently.